providers/os/connection/snapshot/partition.go

```diff
@@ -21,6 +22,17 @@ type PartitionInfo struct {
 	Uuid string
 	// (optional) MountPoint is the partition mount point
 	MountPoint string
+
+	// (optional) MountOptions are the mount options
+	MountOptions []string
+	// (optional) Bind adjusts the root for FS connection
+	Bind string
+}
+
+type MountPartitionDto struct {
+	*PartitionInfo
+
+	ScanDir *string
```

Comment:
is there a chance this is nil? can we do with representing the absence as an empty string>

Reply:
Also, nil check is in place at the MountP side